src/core/router/Routes.js

```diff
@@ -29,6 +29,9 @@ const ORGS :'/orgs' = '/orgs';
 // $FlowFixMe - ignoring flow because I prefer the code hints to show the value
 const ORG :'/orgs/:organizationId' = `${ORGS}/${ORG_ID_PARAM}`;
 
+// $FlowFixMe - ignoring flow because I prefer the code hints to show the value
+const ORG_COLLABORATIONS :'/orgs/:organizationId/collaborations' = `${ORGS}/${ORG_ID_PARAM}/collaborations`;
```

Comment:
No change required, just flagging it for visibility. We're now using both `/collabs` (defined above) and `/collaborations` (defined here). I would prefer if these matched.